README.md

```diff
@@ -218,7 +220,7 @@ Flags:
       --temp                    ON by default: Map the temp folder to a local folder, use --no-temp to disable
       --temp-location=TEMP-LOCATION  
                                 Determine where the temporary work folder 'tgf' inside the docker image is mounted:
-                                
+
```

Comment:
I wondered why you changed this line. Then, then I realized that kingpin inserted an unwanted line here.